src/app/ASKS/ask/index.js

```diff
@@ -671,8 +672,8 @@ const Ask = ({
           clientId={clientId}
         />
       )}
-      {component === 'checkbox' && (
-        <CheckBox.Write
+      {(component === 'single_checkbox' || component === 'checkbox') && (
```

Comment:
It should be { component === 'checkbox' && (... 
instead of 
{(component === 'single_checkbox' || component === 'checkbox') && (